### DISCUSSION TITLE
Cannot get Docker Compatibility to work with Socket for VSCode Containers extension

### DISCUSSION THREAD

**Discussion by JustinGrote** [podman-desktop/podman-desktop#1299] [Q&A]
Hello,

Trying to get podman on Windows working with the vscode containers extension. According to the docs:
https://github.com/containers/podman-desktop/blob/e4e050c0149c8a52615b3100d3b9d7dbda355e34/website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md

I need to swap out my DOCKER_HOST with a podman machine inspection variable. However, inspect shows my podman machine `PodmanSocket` as null. I notice that it also creates a generic docker socket, is this just old documentation?

**Comment by JustinGrote** [comment]
I think I might be tripped up by this brand new (as of the last release) feature?
https://github.com/containers/podman-desktop/pull/1047

**Comment by lstocchi** [comment]
Just for reference https://github.com/containers/podman/issues/16860 I think the doc is not updated for windows. `PodmanSocket` will always be null.

**Comment by JustinGrote** [comment]
Turns out the issue was the newest docker vscode extension is incompatible with podman, see the related issue.
https://github.com/containers/podman/issues/17306
